{"name": "testRefresh[InputData1-Result1]", "status": "passed", "description": "\n        测试登录接口\n        :param InputData:\n        :param Result:\n        :return:\n        ", "attachments": [{"name": "log", "source": "095a365b-dc9b-4065-a94b-f032830bb34d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "InputData", "value": "{'token': '5465465a4d65a4d65sa4adsada'}"}, {"name": "Result", "value": "{'code': 104, 'msg': '登录凭证已失效，请重新登录'}"}], "start": 1619511765864, "stop": 1619511765899, "uuid": "0c8adf85-6152-47eb-a249-a4e397070c3d", "historyId": "645c45bade2919b9972ab343100ee3db", "testCaseId": "da28d27f5a8a935bd1b1ab9a4e6becb4", "fullName": "Testcases.test_02_Refresh.TestUserClass#testRefresh", "labels": [{"name": "feature", "value": "用户相关接口"}, {"name": "story", "value": "刷新token接口"}, {"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('Clean_Tmp')"}, {"name": "parentSuite", "value": "Testcases"}, {"name": "suite", "value": "test_02_Refresh"}, {"name": "subSuite", "value": "TestUserClass"}, {"name": "host", "value": "PC-20200325KZTD"}, {"name": "thread", "value": "10256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcases.test_02_Refresh"}]}